### ISSUE TITLE
simpleplayer fails with error on VideoOutput creation

### ISSUE THREAD

**Issue by xivol** [wang-bin/QtAV#896]
QtAV master built on Windows  with QtCreator 5.6.2 MINGW 4.9.2 32bit. Installed with sdk_install.bat.
I created simpleplayer based widget and that code:
`m_vo = new VideoOutput(this);`
gives me this:
`QPixmap: Must construct a QGuiApplication before a QPixmap`

What am I doing wrong?

**Comment by wang-bin** [comment]
simpleplayer example?

**Comment by xivol** [comment]
Yes, simpleplayer example PlayerWindow widget constructor. When I comment VideoOutput, widget works just fine.
